compile.sh

```diff
@@ -106,12 +106,14 @@ install_hurd() {
    mkdir -p "$HURD_SRC".obj &&
    cd "$HURD_SRC".obj &&
    rm -f config.cache cnfig.status &&
-   ../$HURD_SRC/configure \
+   AR="${ROOT}/bin/${TARGET}-ar" ../$HURD_SRC/configure \
       --build="$HOST" \
       --host="$TARGET" \
       --prefix="$SYS_ROOT" \
       --without-parted \
       --disable-profile &&
+   touch ../$HURD_SRC/libfshelp-tests/BROKEN &&
+   #avoid compile error
```

Comment:
See https://lists.gnu.org/archive/html/bug-hurd/2019-12/msg00008.html , the patch has to be adjusted a bit to be accepted upstream though